doc/tutorials/12-constant_pH/si_units.py

```diff
@@ -25,9 +25,9 @@ def convert(self, from_value, from_unit, to_unit, exponent=1.0):
         if (from_unit == "nm" and to_unit == "sigma"):
             return from_value / self.sigma_to_nm
         if (from_unit == "mol/L" and to_unit == "N/sigma^3"):
-            return from_value * self.conc_N_sigma_to_mol_L
-        if (to_unit == "mol/L" and from_unit == "N/sigma^3"):
             return from_value / self.conc_N_sigma_to_mol_L
+        if (to_unit == "mol/L" and from_unit == "N/sigma^3"):
+            return from_value * self.conc_N_sigma_to_mol_L
```

Comment:
Does this now return that in order to convert from 1/sigma^3 (with sigma=0.355nm) to mol/l, I need to multiply with roughly 37.1? Just checking for a result I know...

Reply:
To get the answer, just add the following line anywhere after initialization of the SI_units class:
`print(SI.convert(1.0,'N/sigma^3','mol/L'))`

Result: 37.11624488606752

Reply:
Great! That s easy.

Reply:
I will soon commit a different approach, that uses the pint package and internally checks for correct dimensionality.